Content.Server/Radio/Components/RadioJammerComponent.cs

```diff
@@ -9,12 +9,25 @@ namespace Content.Server.Radio.Components;
 [Access(typeof(JammerSystem))]
 public sealed partial class RadioJammerComponent : Component
 {
-    [DataField("range"), ViewVariables(VVAccess.ReadWrite)]
-    public float Range = 8f;
+    [DataDefinition]
+    public partial struct RadioJamSetting
+    {
+        [DataField(required: true)]
+        public float Wattage;
 
-    /// <summary>
-    /// Power usage per second when enabled
-    /// </summary>
-    [DataField("wattage"), ViewVariables(VVAccess.ReadWrite)]
-    public float Wattage = 2f;
+        [DataField(required: true)]
+        public float Range;
+
+        [DataField(required: true)]
+        public LocId Message = string.Empty;
+
+        [DataField(required: true)]
+        public LocId Name = string.Empty;
+    }
+
+    [DataField(required: true)]
+    public List<RadioJamSetting> Settings = new();
+
+    [DataField("selectedPowerLevel"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
replace with just [DataField]

Reply:
Like this?

```
    [DataField("selectedPowerLevel")]
    public byte SelectedPowerLevel = 1;
```

Reply:
no just [DataField]

the name is automatically camel cased

Reply:
Fixed it! Sorry for all the mistakes :(